vimpager

```diff
@@ -24,11 +24,13 @@ done
 project_dir="`dirname \"$link\"`"
 # END OF FIND REAL PARENT DIRECTORY
 
-. "$project_dir""/inc/prologue.sh"
-
 version="$(cd "$project_dir"; git describe) (git)"
+runtime="$project_dir"
+vimcat="$project_dir/vimcat"
+
+. "$project_dir""/inc/prologue.sh"
```

Comment:
I think we can remove the two double quotes in the middle. `. "$project_dir/inc/prologue.sh"` should be fine. Or is the original bourne shell strange in this regard?